services/121-service/examples/fsp-anonymous1.json

```diff
@@ -0,0 +1,13 @@
+{
+  "fsp": "Bank A",
+  "attributes": [
+    {
+      "name": "idNumber",
+      "label": {
+        "en": "What is your national ID, so we can transfer money to your bank account"
+      },
+      "answerType": "text",
```

Comment:
This should be a number/numeric, it seems? Or can it contain letters as well?

Reply:
I don't know what it looks like. Seems very probable that a national id 'number' can contain a letters as well.

Reply:
Maybe calling it `nationalId` is more 'inout-type-agnostic'. ;)

But maybe that will clash with some pan-national-IDs, (like EU-wide used numbers)

Reply:
Maybe personalId?